turbo.json

```diff
@@ -30,6 +30,9 @@
     "@shopify/create-hydrogen#build": {
       "dependsOn": ["@shopify/cli-hydrogen#build"]
     },
+    "@shopify/hydrogen#build": {
+      "dependsOn": ["@shopify/hydrogen-react#build"]
+    },
```

Comment:
hydrogen requires hydrogen-react to build first

Reply:
For the types?

Reply:
For everything; hydrogen re-exports things from hydrogen-react, so that will only work if hydrogen-react has been built first.